REST_API_DOCS.md

```
@@ -0,0 +1,1623 @@
+  "error": {
+    "code": "BAD_REQUEST",
+    "message": "잘못된 요청 형식입니다."
+  }
+}
+```
+
+---
+
+### **특정 회원 조회**
+
+**GET** `/users/{id}` 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)